### PR TITLE
Afrikaans Translation

### PR DESCRIPTION
**Pull request by Raithlin**
Afrikaans Translation

**Resolved / Related Issues**
Items resolved / related issues by this PR.

**Details of Changes**
Add details of changes here.
- Added Afrikaans translation

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: Files.Launcher/Win32API.cs [hunk 1]
```diff
@@ -21,553 +21,553 @@
...
+				return new Win32Window() { Handle = new IntPtr(hwnd) };
+			}
+		}
+
+		// Get information from recycle bin.
+		[DllImport(Lib.Shell32, SetLastError = false, CharSet = CharSet.Auto)]
+		public static extern int SHQueryRecycleBin(string pszRootPath,
+			ref SHQUERYRBINFO pSHQueryRBInfo);
+	}
 }
```
Comment (yaichenbaum) [inline Files.Launcher/Win32API.cs:573]: Can you revert the changes to this file?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files.Launcher/Win32API.cs (hunk 1 above)
Comment (Raithlin) [reply Files.Launcher/Win32API.cs]: I couldn't rollback for some reason, but I took the file from the main branch and overwrote it, then committed the change. Does that make a difference on your side?
